modules/misc/qt.nix

```
@@ -141,6 +167,18 @@ in {
     qt.style.package = mkIf (cfg.style.name != null)
       (mkDefault (stylePackages.${toLower cfg.style.name} or null));
 
+    qt.pluginPath =
+      builtins.map (prefix: "${config.home.profileDirectoty}/${prefix}") [
```

The map function is always available:

```suggestion
      map (prefix: "${config.home.profileDirectoty}/${prefix}") [
```